style.css

```diff
@@ -496,3 +501,605 @@ button:active {
+  }
+
+  .cardsub {
+    list-style: none;
+    display: flex;
+    padding-left: 0;
+    justify-content: space-between;
+    justify-self: left;
+    height: 24px;
+    width: 203px;
+  }
+
+  .list {
+    box-sizing: border-box;
+    background-color: #ebebff;
+    color: #6070ff;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 16px;
+    border-radius: 8px;
+    justify-content: space-between;
+  }
+
+  p {
+    height: 50px;
+    width: 490px;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #344563;
+    margin-top: 24px;
+  }
+
+  .c2 {
+    margin-top: 48px;
+  }
+
+  button {
+    height: 48px;
+    width: 124px;
+    box-sizing: border-box;
+    font-size: 17px;
+    color: #396df2;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 24px;
+    border-radius: 8px;
+    padding: 12px;
+    border: 1px solid #6070ff;
+    margin-top: 12px;
+    background-color: #fff;
+  }
+
+  button:hover {
+    color: #fff;
+    background-color: #6070ff;
+
+    /* transition-property: all;
+    transition: 0.2s ease-in-out; */
+  }
+
+  button:active {
+    color: #fff;
+    background-color: #2230d2;
+    border: 1px solid #2230d2;
+  }
+
+  .personal-info {
+    /* display: grid; */
+    margin-top: 54px;
+    width: 100%;
+    margin-left: 0;
+    margin-right: 0;
+    border-radius: 40px 40px 30px 10px;
+  }
+
+  .about-me {
+    display: flex;
+    width: 100%;
+    height: 924px;
+    box-sizing: border-box;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    padding-left: 24px;
+    padding-right: 24px;
+    padding-top: 24px;
+    padding-bottom: 24px;
+    background-color: #fff;
+    border-radius: 0 3px;
+  }
+
+  .dt-about-me {
+    display: flex;
+    flex-direction: column;
+    width: 447px;
+    padding: 24px;
+    height: 392px;
+  }
+
+  .dt-about-me-languages {
+    display: flex;
+    padding: 24px;
+    width: 684px;
+    height: 380px;
+  }
+
+  .dt-language-types {
+    display: flex;
+    flex-direction: row;
+    width: 414px;
+  }
+
+  .about-me h2 {
+    height: 101px;
+    width: auto;
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: left;
+    color: #172b4d;
+  }
+
+  .about-me p {
+    height: 144px;
+    width: auto;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #344563;
+  }
+
+  .social-container {
+    display: flex;
+    flex-direction: column;
+    width: auto;
+    height: 80px;
+  }
+
+  .social-media1 ul {
+    display: flex;
+    justify-content: space-evenly;
+    height: 20px;
+    width: 170px;
+    margin-left: 0;
+    padding-left: 0;
+  }
+
+  .social-media1 ul li {
+    list-style: none;
+    color: #505f79;
+    margin-left: 0;
+  }
+
+  .social-media1 ul li a {
+    /* list-style: none; */
+    color: #505f79;
+  }
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly fix it. :writing_hand: